test/cli/test_cli_functions.sh

```diff
@@ -634,9 +634,61 @@ function account_cycle {
     account_reset_password "admin@noobaa.io"
     # testing nsfs accounts
     # account_nsfs_cycle TODO re-enable.
+    # update account default resource
+    account_update ""
+    # update crd account default resource
+    account_update "crd"
     echo_time "✅  noobaa account cycle is done"
 }
 
+function account_update {
+    local crd=${1}
+    echo_time "💬  Checking for ${crd} account updation"
+    local res="backing-store-upd"
+    if [ "$(kuberun get backingstore | grep -w ${res} | wc -l)" != "1" ]
+    then
+        echo_time "💬  Creating backingstore ${res}"
+        test_noobaa backingstore create pv-pool ${res} --num-volumes 1 --pv-size-gb 16
+    fi
+    if [ "${crd}" == "" ]
+    then
+        local name=$(test_noobaa api account list_accounts {} -o json | jq -r '.accounts[0].email')
+        local prev=$(test_noobaa api account list_accounts {} -o json | jq -r '.accounts[0].default_resource')
+        echo_time "💬  Updating ${res} as default_resource for noobaa account"
+        test_noobaa account update ${name} --new_default_resource=${res}
+        local curr=$(test_noobaa api account list_accounts {} -o json | jq -r '.accounts[0].default_resource')
+        if [ "${prev}" == "${curr}" ]
+        then
+            echo_time "❌ Looks like account not updated, Exiting"
+            exit 1
+        else
+            echo_time "✅  Update account default_resource successful"
+            test_noobaa account update ${name} --new_default_resource=${prev}
```

Comment:
```suggestion
            echo_time "✅  Update account default_resource successful"
            echo_time "💬  Updating ${prev} as default_resource for noobaa account"
            test_noobaa account update ${name} --new_default_resource=${prev}
```
We should also also verify that it worked.

Reply:
Added check.